nipoppy/tabular/base.py

```diff
@@ -91,6 +91,8 @@ def load(cls, fpath: StrOrPathLike, validate=True, **kwargs) -> Self:
                 "This function does not accept 'dtype' as a keyword argument"
                 ". Everything is read as a string and (optionally) validated later."
             )
+        if "sep" not in kwargs:
+            kwargs["sep"] = "\t"
         df = cls(pd.read_csv(fpath, dtype=str, **kwargs))
```

Comment:
Consider checking that the TSV is actually a TSV and not a CSV: https://github.com/neurobagel/bagel-cli/blob/5d15125cdce8c14357d159e3b377c0b0ec7ac8b1/bagel/pheno_utils.py#L44-L84